tests/allure/src/main/java/org/mule/test/allure/AllureConstants.java

```diff
@@ -95,6 +95,16 @@ interface UndeploymentFailureStory {
       String UNDEPLOYMENT = "Undeployment";
 
     }
+
+    interface SupportedJavaVersions {
+
+      String JAVA_VERSIONS_IN_DEPLOYABLE_ARTIFACT =
+          "Supported Java Versions are reflected in the ApplicationModel or DomainModel";
+
+      String ENFORCE_DEPLOYABLE_ARTIFACT_JAVA_VERSION =
+          "Validate that all registered extensions support the Java version Mule is running on";
```

Comment:
Similarly, it looks like this description should be adapted to deployable artifacts and not extensions.